File: xMBSv1/CodeFirstMembership/Entities/Role.vb
Imports System.ComponentModel.DataAnnotations

Public Class Role

    <Key()>
    Public Overridable Property RoleId As Guid

    <Required()>
    Public Overridable Property RoleName As String

    Public Overridable Property Description As String

    Public Overridable Property Users As ICollection(Of User)

End Class

File: trunk/xMBSv2/xMBSv2/Models/Entities/Role.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace xMBSv2.Models.Entities
{
    using System.ComponentModel.DataAnnotations;

    public class Role
    {
        [Key]
        public virtual Guid RoleId { get; set; }

        [Required]
        public virtual string RoleName { get; set; }

        public virtual string Description { get; set; }

        public virtual ICollection<User> Users { get; set; }
    }
}